.appveyor.yml

```diff
@@ -4,7 +4,7 @@ skip_commits:
 environment:
 
   CONDA_INSTALL_LOCN: "C:\\Miniconda36-x64"
-  CONFIG: azure-win-64
+  CONFIG: win64
```

Comment:
Are these all temporary changes?

Reply:
No, I needed the $CI variable and these name changes were a mistake. They are actually used in https://github.com/conda-forge/staged-recipes/blob/master/.ci_support/build_all.py#L78 and somehow it was broken.